adagrams/game.py

```diff
@@ -1,11 +1,91 @@
+import random
+
 def draw_letters():
-    pass
+    pool_count = 98 # number of letters in pool overall
+    drawn_letters = []
+    letter_pool = [] # list of letters available draw from
+    letter_pool_amounts = {
+        'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12,
+        'F': 2, 'G': 3, 'H': 2, 'I': 9, 'J': 1,
+        'K': 1, 'L': 4, 'M': 2, 'N': 6, 'O': 8,
+        'P': 2, 'Q': 1, 'R': 6, 'S': 4, 'T': 6,
+        'U': 4, 'V': 2, 'W': 2, 'X': 1, 'Y': 2, 'Z': 1
+    }
+
+    # creates a list of individual letters to draw from
+    for letter, number in letter_pool_amounts.items():
+        for x in range(number):
+            letter_pool.append(letter)
+    # draws 10 random letters and returns them as a list
+    for x in range(10):
+        letter_index = random.randint(0, pool_count - 1)
+        drawn_letters.append(letter_pool.pop(letter_index))
+        pool_count -= 1
+    return drawn_letters
```

Comment:
Great logic! One stylistic note: in the loops on lines 17 and 20 we never use the `x` variable. To make it clear that this variable is unneeded we can use an underscore instead: `for _ in range(10)`:. This is a common idiom in Python when we want to show we don't care about the value of a variable.